dae/dae/genomic_resources/cli.py

```diff
@@ -789,12 +782,12 @@ def cli_browse(cli_args=None):
             if definition_path is not None \
             else DEFAULT_DEFINITION
 
-    if args.print_grr:
-        if definition_path is not None:
-            print("Working with GRR definition:", definition_path)
-        else:
-            print("No GRR definition found, using the DEFAULT_DEFINITION")
-        pprint.pprint(definition)
+
+    if definition_path is not None:
+        print("Working with GRR definition:", definition_path)
+    else:
+        print("No GRR definition found, using the DEFAULT_DEFINITION")
+    print(yaml.dump(definition))
```

Comment:
Always prefer `safe_dump` instead of `dump`.  Consider using `sort_keys=False` for keeping the GRR definition close to the original.